akka-stream/src/main/resources/reference.conf

```diff
@@ -111,10 +113,9 @@ akka {
       //#stream-ref
     }
 
-    # Fully qualified config path which holds the dispatcher configuration
-    # to be used by ActorMaterializer when creating Actors for IO operations,
-    # such as FileSource, FileSink and others.
-    blocking-io-dispatcher = "akka.stream.default-blocking-io-dispatcher"
+    # Deprecated, use akka.stream.materializer.blocking-io-dispatcher, this setting
+    # was never applied because of bug #24357
+    blocking-io-dispatcher = "<deprecated>"
```

Comment:
I couldn't reach it from the materializer settings when down on a nesting level below materializer, and additionally, it logically belongs to the materializer settings.